.github/workflows/cmake.yml

```diff
@@ -43,18 +46,19 @@ jobs:
     - name: Install Linux Dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install libproc2-dev libopencv-dev libopencv-dev wine64-tools
-        sudo apt-get install qttools5-dev qtbase5-dev libqt5serialport5-dev qtbase5-private-dev
-      if: matrix.os == 'ubuntu-latest'
+        sudo apt-get install libprocps-dev libopencv-dev wine64-tools
+        wget -q https://github.com/microsoft/onnxruntime/releases/download/v${{matrix.onnx_version}}/${{matrix.onnx_prefix}}-${{matrix.onnx_version}}.tgz
+        tar xzf ${{matrix.onnx_prefix}}-${{matrix.onnx_version}}.tgz
+      if: contains(matrix.os, 'ubuntu')
 
     - name: Install Qt
-      uses: jurplel/install-qt-action@v3
+      uses: jurplel/install-qt-action@v4
       with:
-        archives: qtbase qtimageformats qtgamepad qttools qtserialport qtmultimedia
-      if: matrix.os != 'ubuntu-latest'
+        version: 5.15
+        archives: icu qtbase qtimageformats qtgamepad qttools qtserialport qtmultimedia
 
     - name: Configure
-      run: ${{matrix.cmake}} -GNinja -S ${{github.workspace}}/ -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DQt5_DIR=${{env.Qt5_DIR}} -DQt5Gui_DIR=${{env.Qt5_DIR}}/lib/cmake/Qt5Gui
+      run: ${{matrix.cmake}} -GNinja -S ${{github.workspace}}/ -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DCMAKE_PREFIX_PATH=${{env.Qt5_DIR}} -DONNXRuntime_DIR=${{matrix.onnx_prefix}}-${{matrix.onnx_version}}
```

Comment:
Don't' touch `Qt5_DIR`. It's not even correct to be using `CMAKE_PREFIX_PATH` for just one dependency out of several. The bit about `Qt5Gui_DIR` maybe could even be removed.

I suppose using the official package is better than relying on some arbitrary PPA.